src/frontend/src/components/footer.ts

```diff
@@ -34,7 +34,7 @@ export const footer = html`<footer class="l-footer">
   ${navigationLink({
     labelText: "FAQ",
     id: "faqLink",
-    url: "/faq",
+    url: "https://support.dfinity.org/hc/en-us/sections/8730568843412-Internet-Identity",
```

Comment:
do you need to extend `navigationLink` with `target=_blank` and `rel=noopener noreferrer`?

Reply:
Good point, spotted another link that was missing it